Misc/NEWS.d/next/Library/2024-11-10-18-14-51.gh-issue-104745.zAa5Ke.rst

```diff
@@ -0,0 +1,5 @@
+Limit starting a patcher (from :func:`unittest.mock.patch`,
+:func:`unittest.mock.patch.object` or :func:`unittest.mock.dict`) more than
+once without stopping it. Previously, this would cause an
+:exc:`AttributeError` if the patch stopped more than once after this, and
+would also disrupt the original patched object.
```

Comment:
NEWS entries should be pretty short, and `unittest.mock.dict` is undocumented, so we need to prefix it with `!`:
```suggestion
Limit starting a patcher (from :func:`unittest.mock.patch`,
:func:`unittest.mock.patch.object` or :func:`!unittest.mock.dict`) more than
once without stopping it.
```

Reply:
It's actually `unittest.mock.patch.dict`: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict

Reply:
Oh! Thank you!